core/domain/skill_services.py

```diff
@@ -24,8 +24,10 @@
 from core.platform import models
 import feconf
 
-(skill_models, user_models, question_models) = models.Registry.import_models(
-    [models.NAMES.skill, models.NAMES.user, models.NAMES.question])
+(skill_models, user_models, question_models, base_models) = (
```

Comment:
Why do you need base_models? It seems unused.

Make sure to check the "Files Changed" of your PR carefully and ensure that only the stuff you want to include is present.

Reply:
Done.